css.css

```diff
@@ -54,7 +54,7 @@ transition-delay:0,2s;
   background-image: url("img/Bitcoin-Emblem.png");
   background-repeat: no-repeat;
   background-position: center;
-  background-size: 1920px 1080px;
+  background-size: 1440px 810px;
```

Comment:
background size "contain"
width "auto"
Zo zorg je er voor dat het op elk scherm goed zichtbaar is.